.env.sample

```diff
@@ -2,7 +2,7 @@
 # in docker-compose.yml. Rename this file to `.env` and then uncomment and set any variable below.
 
 # Supported ethereum network (for ex: mainnet, goerli, sepolia, ropsten, gnosis) .
-#ETH2_NETWORK=
```

Comment:
rather follow the cdvn repo pattern that uses defaults so using .env is optional not required.

Reply:
done